packages/tunnel/src/commands/deploy/index.ts

```diff
@@ -0,0 +1,86 @@
+import { existsSync } from 'node:fs';
+import path from 'node:path';
+
+import { isValidUrl } from '@logto/core-kit';
+import chalk from 'chalk';
+import type { CommandModule } from 'yargs';
+
+import { consoleLog } from '../../utils.js';
+
+import { type DeployCommandArgs } from './types.js';
+import { deployToLogtoCloud } from './utils.js';
+
+const tunnel: CommandModule<unknown, DeployCommandArgs> = {
+  command: ['deploy'],
+  describe: 'Deploy your custom UI assets to Logto Cloud',
+  builder: (yargs) =>
+    yargs.options({
+      auth: {
+        describe: 'Auth credentials of your Logto M2M application. E.g.: <app-id>:<app-secret>',
+        type: 'string',
+      },
+      endpoint: {
+        describe:
+          'Logto endpoint URI that points to your Logto Cloud instance. E.g.: https://<tenant-id>.logto.app/',
+        type: 'string',
+      },
+      path: {
+        alias: ['experience-path'],
+        describe: 'The local folder path of your custom sign-in experience assets.',
+        type: 'string',
+      },
+      resource: {
+        alias: ['management-api-resource'],
+        describe: 'Logto Management API resource indicator. Required if using custom domain.',
+        type: 'string',
+      },
+      verbose: {
+        describe: 'Show verbose output.',
+        type: 'boolean',
+        default: false,
+      },
+    }),
+  handler: async (options) => {
```

Comment:
we may also print the docs link for the help command

Reply:
cool, np

Reply:
updated with `.epilog()` function

Reply:
side topic: i'm actually thinking about to use a shorten url in case we forgot to update here in the future